content/influxdb/clustered/install/configure-cluster.md

```diff
@@ -821,4 +821,36 @@ spec:
 
 {{% /code-placeholders %}}
 
+### Provide a custom Certificate Authority bundle (optional)
+
+If you do not use a private certificate authority, then you can skip this section.
+
+InfluxDB will attempt to make TLS connections to the services it depends on; notably the Postgresql catalog, and the object store. InfluxDB validates the certificates for all of the connections it makes. If you host these services yourself, and you use a private or otherwise not well-known certificate authority to issue certificates to those services, then InfluxDB will not recognize the issuer and will not be able to validate them. To allow InfluxDB to validate these certificates, you must provide it with a pem bundle containing your custom certificate chain.
+
+First, you must create a config map containing your pem bundle. In the example below, `private_ca.pem` is a pem formatted certificate bundle file. It should be provided to you by your certificate authority administrator. Note that this is *not* the certificate that InfluxDB uses to host its own TLS endpoints. This bundle establishes a chain of trust for the external services that InfluxDB depends on.
```

Comment:
```suggestion
InfluxDB attempts to make TLS connections to the services it depends on; notably
the [Catalog](/influxdb/clustered/reference/internals/storage-engine/#catalog),
and the [Object store](/influxdb/clustered/reference/internals/storage-engine/#object-store).
InfluxDB validates the certificates for all of the connections it makes.

**If you host these services yourself and you use a private or otherwise not
well-known certificate authority to issue certificates to theses services**, 
InfluxDB will not recognize the issuer and will be unable to validate the certificates.
To allow InfluxDB to validate these certificates, provide a PEM certificate
bundle containing your custom certificate authority chain.
```